RecoEgamma/EgammaPhotonAlgos/src/ConversionSeedFinder.cc

```diff
@@ -1,5 +1,7 @@
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "Geometry/TrackerGeometryBuilder/interface/TrackerGeometry.h"
+#include "Geometry/CommonDetUnit/interface/GlobalTrackingGeometry.h"
```

Comment:
I do not see GlobalTrackingGeometry used in this .cc file
also, the contents of Geometry/CommonDetUnit/interface/GlobalTrackingGeometry.h file is just 
```#include "Geometry/CommonTopologies/interface/GlobalTrackingGeometry.h"```

it seems like this second include is not needed here or should be replaced by somthing that's actually used

Reply:
yes, it can be removed. Thanks!